tests/thread/host/host.cpp

```diff
@@ -286,10 +286,15 @@ size_t host_tcs_out_thread_count()
     return g_tcs_out_thread_count;
 }
 
+int test_errno_multi_threads_sameenclave(oe_enclave_t* enclave);
+int test_errno_multi_threads_diffenclave(
+    oe_enclave_t* enclave1,
+    oe_enclave_t* enclave2);
```

Comment:
Could you move these forward declarations to the top of the file? Along with `test_readers_writer_lock` which isn't part of this PR but I would prefer they all go together.